src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+for (let i = 0; i < table.rows.length; i++) {
```

Comment:
Code Quality Issue: It's generally not recommended to directly iterate over HTML collections like `table.rows`, because they are live collections. This means that if you modify the collection while iterating over it (like by adding or removing elements), it can lead to unexpected behavior. It would be better to convert it to an array first using `Array.from()` or the spread operator (`...`).